clients/cloud/java/README.md

```diff
@@ -149,24 +151,24 @@ Note that your VPC must be able to connect to the Confluent Cloud Schema Registr
 
 1. As described in the [Confluent Cloud quickstart](https://docs.confluent.io/current/quickstart/cloud-quickstart/schema-registry.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud), in the Confluent Cloud GUI, enable Confluent Cloud Schema Registry and create an API key and secret to connect to it.
 
-2. Verify your Confluent Cloud Schema Registry credentials work from your host. In the output below, substitute your values for `<SR API KEY>`, `<SR API SECRET>`, and `<SR ENDPOINT>`.
+2. Verify your Confluent Cloud Schema Registry credentials work from your host. In the output below, substitute your values for `<SR_API_KEY>`, `<SR_API_SECRET>`, and `<SR_ENDPOINT>`.
 
     ```shell
     # View the list of registered subjects
-    $ curl -u <SR API KEY>:<SR API SECRET> https://<SR ENDPOINT>/subjects
+    $ curl -u <SR_API_KEY>:<SR_API_SECRET> https://<SR_ENDPOINT>/subjects
 
-    # Same as above, as a single bash command to parse the values out of $HOME/.ccloud/config
-    $ curl -u $(grep "^schema.registry.basic.auth.user.info" $HOME/.ccloud/config | cut -d'=' -f2) $(grep "^schema.registry.url" $HOME/.ccloud/config | cut -d'=' -f2)/subjects
+    # Same as above, as a single bash command to parse the values out of example.config
+    $ curl -u $(grep "^schema.registry.basic.auth.user.info" $HOME/examples/clients/cloud/java/example.config | cut -d'=' -f2) $(grep "^schema.registry.url" $HOME/examples/clients/cloud/java/example.config| cut -d'=' -f2)/subjects
     ```
 
-3. Add the following parameters to your local Confluent Cloud configuration file (``$HOME/.ccloud/config``). In the output below, substitute values for `<SR API KEY>`, `<SR API SECRET>`, and `<SR ENDPOINT>`.
+3. Add the following parameters to your local Confluent Cloud configuration file (``$HOME/examples/clients/cloud/java/example.config``). In the output below, substitute values for `<SR_API_KEY>`, `<SR_API_SECRET>`, and `<SR_ENDPOINT>`.
```

Comment:
For reasons stated earlier:

1. Simplify path to just example.config
2. Hyperlink